.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -26,6 +26,7 @@ about: Beschrijving van een feature/user story
 - [ ] de DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
 - [ ] eventueel gemaakte ontwerp keuzes zijn gedocumenteerd
 - [ ] er zijn geen (nieuwe) conflicten met ontwerp keuzes in [BIP](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/docs/design_decisions.md)
+- [ ] de gewenste (aanvulling op de) functionaliteit (van de API's) is geanalyseerd en beschreven.
 
```

Comment:
Wat hier (DoD) m.i. nog mist is:
- de API-documentatie is bijgewerkt (functioneel en technisch).
- gedocumenteerd is welke wijzigingen zijn doorgevoerd in de API's a.g.v. de user-story.
- in de user-story wordt, t.b.v. de review, verwezen naar de desbetreffende documentatie.